plugins/node/opentelemetry-instrumentation-mongodb/package.json

```diff
@@ -58,7 +58,7 @@
     "@types/node": "16.11.21",
     "gts": "3.1.0",
     "mocha": "7.2.0",
-    "mongodb": "3.6.11",
+    "mongodb": "4.6.0",
```

Comment:
This is deliberately set to version 3 as it is what we run in CI when invoking `npm run test`:

```
    "test": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/**/mongodb-v3.test.ts'",
```

Since we can only run a single version, version 3 was choosen as it is more common